README.md

```diff
@@ -106,6 +106,9 @@ _alias: t.deepEquals_
 
 Assert that `a` and `b` are deeply equal.
 
+## Test
+* [Runt tests in browser](https://hakanols.github.io/teston/test/index.html)
```

Comment:
Could this be https://testonjs.github.io/teston/test/index.html, I can activate it

Reply:
It should be changed in the last commit. As you writer, you have to activate pages under the repository settings.